.github/workflows/generated.yml

```diff
@@ -28,12 +32,16 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: jetify-com/devbox-install-action@v0.11.0
-        with:
-          enable-cache: true
+      - uses: jdx/mise-action@v2
+
+      - name: Install dependencies
+        run: sudo apt -y install libwebp-dev
```

Comment:
Is this package still needed now that we use a go native webp library? (16b33770dfed5eebb2699d8b02bd53332ed686d8)